Ros2 enhancement add imu output

anscipione opened the pull request:
- protocol adapted to firmware 5.2
- fixed bug that prevented the reading of data from the serial
- added reading of imu data
- added program to enumerate vescs based on their uuid
- added possibility to command more vesc through canForward

anscipione committed:
adeguato a FW 5.2 ed introdotta lettura IMU

anscipione committed:
risolto bug lettura ed aggiunta regola per creazione device

JWhitleyWork commented:
@anscipione Thank you very much for the PR! However, this is a very extensive overhaul of the code (almost 2,000 lines of changes) which is a bit hard to review all-at-once. Would you mind splitting this into two PRs? Even if one depends on the other, it will be much easier to review them separately.

anscipione committed:
reolved data lost sync

reduced timeout from 10ms to 1ms

anscipione commented:
ok I will try to divide the changes by arguments.

1. bug fixing of serial comunication.
2. protocol adapted to firmware 5.2 (this is the change that requires the biggest changes to the code I think )
3. added reading of imu data 
4. added possibility to command more vesc through canForward 
5. added program to enumerate vescs based on their uuid

can this division fit for you?

unfortunately they are all dependent on each other so I will have to proceed in order.

JWhitleyWork commented:
Yes, that should work fine. I'll do my best to respond to the PRs quickly.
